### PR TITLE
Show users who has authored and approved changes

### PR DESCRIPTION
**Pull request by tadast**
screen shot 2015-11-16 at 16 42 03

### CODE REVIEW

File: spec/features/guide_publishing_flow_spec.rb [hunk 1]
```diff
@@ -155,7 +155,7 @@
         edition = Generators.valid_edition(state: "review_requested", title: "Standups")
         guide = Guide.create!(latest_edition: edition, slug: "/service-manual/something")
 
-        reviewer = User.new(name: "Some User")
+        reviewer = User.new(name: "Keanu Reviews")
```
Comment (KushalP) [inline spec/features/guide_publishing_flow_spec.rb]: :ok_hand:

File: app/models/edition.rb [hunk 2]
```diff
@@ -4,7 +4,7 @@ class Edition < ActiveRecord::Base
   belongs_to :guide, touch: true
   belongs_to :user
 
-  has_many :approvals
+  has_one :approval
```
Comment (AndrewVos) [inline app/models/edition.rb]: How does this work without migrations? Does it take the last approval?

**Comment by AndrewVos** [comment]
I'm not sure I like that `has_one`.

File: app/views/editions/show.html.erb [hunk 3]
```diff
@@ -49,6 +49,24 @@
         <%= @edition.updated_at %>
       </div>
     </div>
+
+    <div class="panel panel-default">
+      <div class="panel-heading">
+        Authors
```
Comment (KushalP) [inline app/views/editions/show.html.erb]: If this is a heading, why isn't it using one of the respective `<h*>` elements?

File: app/models/edition.rb (hunk 2 above)
Comment (tadast) [inline app/models/edition.rb]: We only had one approval per edition already, so it works nicely without migrating any data.

File: app/views/editions/show.html.erb (hunk 3 above)
Comment (tadast) [inline app/views/editions/show.html.erb]: I followed examples on http://getbootstrap.com/components/#panels. These are very likely to change once Mike goes through the UI (soon), so I wouldn't worry about it too much.

**Comment by tadast** [comment]
@AndrewVos apart from disliking `has_one` do you have any concrete concerns about it?

**Comment by AndrewVos** [comment]
Yeah, the overloaded use of `has_one` is less explicit than I would like it to be. Consider adding `approved_by_id => User`

**Pull request merged**